src/yaml.zig

```diff
@@ -190,6 +197,17 @@ pub const Value = union(enum) {
                 return Value{ .float = float };
             }
 
+            const isTruthy = std.mem.eql(u8, raw, "true");
+            const isFalsy = std.mem.eql(u8, raw, "false");
```

Comment:
I am no YAML expert but I think we should handle all possible variants for a boolean:

```
y|Y|yes|Yes|YES|n|N|no|No|NO|
true|True|TRUE|false|False|FALSE|
on|On|ON|off|Off|OFF
```

taken from https://stackoverflow.com/questions/53648244/specifying-the-string-value-yes-in-yaml#53648738

Reply:
Okay, I will look into this

Reply:
I checked the yaml specification and it seems y|Y|yes|Yes|YES|n|N|no|No|NO is a backwards-compatible support but it will be nice to have for this project